Makefile

```diff
@@ -88,16 +88,45 @@ RESTORE_DIR ?= .latest.restore
 
 pg_dump:
 	@PGDATABASE=${PROD_PGDATABASE} PGHOST=${PROD_PGHOST} PGPORT=${PROD_PGPORT} PGUSER=${PROD_PGUSER} PGPASSWORD=${PROD_PGPASSWORD} \
-			psql -c ";" || { \
-				echo "\n\n### Your connexion to the production database failed. Is the port blocked by a firewall ? ###\n" ; \
-				false ; \
-			}
+		psql -c ";" || { \
+			echo "\n\n### Your connexion to the production database failed. Is the port blocked by a firewall ? ###\n" ; \
+			false ; \
+		}
 	rm -rf $(DUMP_DIR)
 	@echo "\n\n### Dumping the production database on your machine, check your disk size and connection speeds ! ###\n"
 	PGDATABASE=${PROD_PGDATABASE} PGHOST=${PROD_PGHOST} PGPORT=${PROD_PGPORT} PGUSER=${PROD_PGUSER} PGPASSWORD=${PROD_PGPASSWORD} \
-		   time pg_dump --format=directory --no-owner --no-acl --verbose --jobs=4 \
-		   --exclude-table="lh_*" --exclude-table="yp_*" --exclude-table="z_*" \
-		   --file=$(DUMP_DIR)
+		time pg_dump --format=directory --no-owner --no-acl --verbose --jobs=4 \
+		--table="fluxIAE_*" \
+		--table="c1_*" \
+		--table="candidats_v0" \
+		--table="candidatures" \
+		--table="cap_campagnes" \
```

Comment:
why not `cap_*`, `suivi_*` et `*_v0` tant qu'à faire

Reply:
+1 pour `cap_*` et `suivi_*` si ça ne prend effectivement que les tables nécessaires.
Mais plutôt -0 pour `*_v0` car c'est un suffixe, qu'il est trop générique, et qu'idéalement il faudrait avoir un préfixe emplois car si jamais on faisait une table en `_v1` alors elle ne serais pas prise actuellement.

Reply:
`suivi_*` c'est mort y en a trop
Je regarderai pour `cap_*`